openshift/manifests/0000_30_cluster-api_04_cm.infrastructure-azure.yaml

```diff
@@ -11025,6 +11025,129 @@ data:
+      paramRef:
+        name: cluster
+        parameterNotFoundAction: Deny
+      policyName: openshift-cluster-api-protect-azurecluster
+      validationActions:
+      - Deny
+    ---
+    apiVersion: admissionregistration.k8s.io/v1
+    kind: ValidatingAdmissionPolicy
+    metadata:
+      name: openshift-cluster-api-protect-azureclusteridentities
+    spec:
+      failurePolicy: Fail
+      matchConstraints:
+        resourceRules:
+        - apiGroups:
+          - infrastructure.cluster.x-k8s.io
+          apiVersions:
+          - '*'
+          operations:
+          - DELETE
+          resources:
+          - azureclusteridentities
+      paramKind:
+        apiVersion: config.openshift.io/v1
+        kind: Infrastructure
+      validations:
+      - expression: '!(oldObject.metadata.name == params.status.infrastructureName)'
+        message: InfraCluster resources with metadata.name corresponding to the cluster
+          infrastructureName cannot be deleted.
+    ---
+    apiVersion: admissionregistration.k8s.io/v1
+    kind: ValidatingAdmissionPolicyBinding
+    metadata:
+      name: openshift-cluster-api-protect-azureclusteridentities
+    spec:
+      matchResources:
+        namespaceSelector:
+          matchLabels:
+            kubernetes.io/metadata.name: openshift-cluster-api
+      paramRef:
+        name: cluster
+        parameterNotFoundAction: Deny
+      policyName: openshift-cluster-api-protect-azureclusteridentities
+      validationActions:
+      - Deny
+    ---
+    apiVersion: admissionregistration.k8s.io/v1
+    kind: ValidatingAdmissionPolicy
+    metadata:
+      name: openshift-cluster-api-protect-azureclustersecrets
+    spec:
+      failurePolicy: Fail
+      matchConstraints:
+        resourceRules:
+        - apiGroups:
+          - ""
+          apiVersions:
+          - '*'
+          operations:
+          - DELETE
+          resources:
+          - secrets
+          resourceNames:
+          - capz-manager-bootstrap-credentials
+          - capz-manager-cluster-credential
+      paramKind:
+        apiVersion: config.openshift.io/v1
+        kind: Infrastructure
+      validations:
+      - expression: '!(oldObject.metadata.name == params.status.infrastructureName)'
+        message: InfraCluster resources with metadata.name corresponding to the cluster
+          infrastructureName cannot be deleted.
+      ---
+      apiVersion: admissionregistration.k8s.io/v1
+      kind: ValidatingAdmissionPolicyBinding
+      metadata:
+        name: openshift-cluster-api-protect-azureclustersecrets
+      spec:
+        matchResources:
+          namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: openshift-cluster-api
+        paramRef:
+          name: cluster
+          parameterNotFoundAction: Deny
+        policyName: openshift-cluster-api-protect-azureclustersecrets
+        validationActions:
+        - Deny
```

Comment:
The indentation of this file is off (too much on the right end side, which is throwing off the parser with:
`E1025 10:54:25.739990 1 controller.go:316] "Reconciler error" err="error during reconcile: error applying CAPI provider \"azure\" components: error getting provider components: error parsing provider component at position 32 to unstructured: error while decoding YAML to runtime object: error while decoding YAML object: yaml: line 29: could not find expected ':'" controller="CapiInstallerController" controllerGroup="config.openshift.io" controllerKind="ClusterOperator" ClusterOperator="cluster-api" namespace="" name="cluster-api" reconcileID="ff9c7798-9de6-4629-a95e-c358aaecf1f2"
E1025 10:54:29.500601 1 kind.go:71] "if kind is a CRD, it should be installed before calling Start" err="failed to get restmapping: no matches for kind \"AzureCluster\" in version \"infrastructure.cluster.x-k8s.io/v1beta1\"" logger="controller-runtime.source.EventHandler" kind="AzureCluster.infrastructure.cluster.x-k8s.io"`